src/Service/Modules/SernosConfigModule.cs

```diff
@@ -12,13 +13,75 @@ public sealed class SernosConfigModule : NancyModule
     {
         private readonly IFacadeService<SernosConfig, string, SernosConfigResource, SernosConfigResource> sernosConfigService;
 
-        public SernosConfigModule(IFacadeService<SernosConfig, string, SernosConfigResource, SernosConfigResource> sernosConfigService)
+        private readonly IFacadeService<SernosTrans, string, SernosTransactionResource, SernosTransactionResource> sernosTransactionService;
+
+        private readonly IFacadeService<SernosCount, string, SernosCountResource, SernosCountResource> sernosCountFacadeService;
+
+        public SernosConfigModule(
+            IFacadeService<SernosConfig, string, SernosConfigResource, SernosConfigResource> sernosConfigService,
+            IFacadeService<SernosTrans, string, SernosTransactionResource, SernosTransactionResource> sernosTransactionService,
+            IFacadeService<SernosCount, string, SernosCountResource, SernosCountResource> sernosCountFacadeService)
         {
             this.sernosConfigService = sernosConfigService;
+            this.sernosTransactionService = sernosTransactionService;
+            this.sernosCountFacadeService = sernosCountFacadeService;
             this.Get("/products/maint/sernos-configs/{name}", parameters => this.GetSernosConfigByName(parameters.name));
             this.Get("/products/maint/sernos-configs/", _ => this.GetSernosConfigs());
             this.Put("/products/maint/sernos-configs/{name}", parameters => this.UpdateSernosConfig(parameters.name));
             this.Post("/products/maint/sernos-configs", _ => this.AddSernosConfig());
+
+            this.Get("/products/maint/serial-number-transactions", _ => this.GetTransCodes());
+            this.Get("/products/maint/serial-number-transactions/{id}", parameters => this.GetTransCode(parameters.id));
+            this.Post("/products/maint/serial-number-transactions", _ => this.AddTransCode());
+            this.Put("/products/maint/serial-number-transactions/{id}", parameters => this.UpdateTransCode(parameters.id));
+
+            this.Get("/products/maint/serial-number-counts", _ => this.GetSernosCounts());
+        }
+
+        private object GetSernosCounts()
+        {
+            var results = this.sernosCountFacadeService.GetAll();
```

Comment:
stylistic preference but i'd just drop the service call in the arg for WithModel

Reply:
Yeah. It's a good call. I slightly annoy myself every time I call a facade service like this but keep doing it for no good reason